### DISCUSSION TITLE
Week start on Monday

### DISCUSSION THREAD

**Discussion by kwbln** [YukiGasai/obsidian-google-calendar#225] [Ideas]
Hi there!
It would be great to be able to change the week start from Sunday to Monday. This has also impact on the calendar week which I use in the title of my daily notes.
Best regards,
Kerstin

**Comment by fjavier10** [comment]
I am also interested in changing obsidian google calendar visualization so that weeks start in Mondays.
Do not find a way to do it in settings for Google Calendar.
Best
Javier
